### PR TITLE
Fix not commiting all .md files for `update_docs`

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a changelog summary in the Pull Request description @KrauseFx</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: fastlane/Fastfile [hunk 1]
```diff
@@ -166,7 +166,10 @@ lane :update_docs do
   template_path = File.expand_path("./assets/render_plugin.md.erb")
   clone_docs do
     docs_path = generate_markdown_docs(target_path: ".")
-    actions_md_path = File.join(docs_path, "actions.md")
+    actions_md_path = File.expand_path(File.join(docs_path, "docs/actions.md"))
```
Comment (taquitos) [inline fastlane/Fastfile]: I'm not understanding why `actions.md` changes to `docs/actions.md`

Comment (KrauseFx) [reply fastlane/Fastfile]: The return value was changed with a previous PR to not return the path to the directory containing `actions.md`, but the directory containing the `docs` folder, containing the `actions.md`. This is due to the fact that we now also copy resources (e.g. screenshots and logos), so we need to access those as well :)

Comment (taquitos) [reply fastlane/Fastfile]: Ahhhh awesome

**Review by taquitos — APPROVED** [review]
🎈🐐

**Pull request merged**